__tests__/app.test.js

```diff
@@ -2,135 +2,188 @@ const app = require("../app");
-        return request(app)
-        .get('/api/articles')
-        .expect(200)
-        .then(({body: {articles}}) =>{
-            articles.forEach((article)=>{
-                expect(article).not.toHaveProperty('body')
-            })
-        })
-    })
-    test('there should be a comment_count property on each arrticle in the array that has a value equal to the number of comments in the comments table that have the same article_id as the article',()=>{
-        return request(app)
-        .get('/api/articles')
-        .expect(200)
-        .then(({body: {articles}}) =>{
-            articles.forEach((article)=>{
-                expect(article).toHaveProperty('comment_count')
-                let testCommentId = 1
-                const comments = require('../db/data/test-data/comments')
-                let testCount =0
-                comments.forEach((comment)=>{if(comment.article_id === testCommentId){testCount++}})
-                    expect(article.comment_count === testCount)
+      });
+  });
+  test("there should be a comment_count property on each arrticle in the array that has a value equal to the number of comments in the comments table that have the same article_id as the article", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        articles.forEach((article) => {
+          expect(article).toHaveProperty("comment_count");
             })
-        })
-    })
-    test('the array should be sorted by date in descending order',()=>{
-        return request(app)
-        .get('/api/articles')
-        .expect(200)
-        .then(({body: {articles}}) =>{
-        expect(articles).toBeSortedBy('created_at', {
-            descending: true,
-          })
-      });
-    })
-})
\ No newline at end of file
+        });
+      });
+  test("the array should be sorted by date in descending order", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toBeSortedBy("created_at", {
+          descending: true,
+        });
+      });
+  });
+});
+describe("/api/articles/:article_id/comments", () => {
+  test("GET:200 responds with an array of comments for the given article_id, each comment should have comment_id, votes, created_at, author, body and article_id properties", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body: { comments } }) => {
+        expect(Array.isArray(comments)).toBe(true);
+        comments.forEach((comment) => {
+          expect(comment).toHaveProperty('comment_id');
+          expect(comment).toHaveProperty('votes');
+          expect(comment).toHaveProperty('created_at');
+          expect(comment).toHaveProperty('author');
+          expect(comment).toHaveProperty('body');
+          expect(comment).toHaveProperty('article_id');
```

Comment:
I would expect you to assert here that each comment also has the article_id of 1 as that is the article you have requested the comments on.